### ISSUE TITLE
Update website's meta tag to include "and Windows" ;)

### ISSUE THREAD

**Issue by shanselman** [docker/kitematic#1263]
<meta property=og:description content="Kitematic - The easiest way to use Docker on Mac.">

Update website's meta tag to include "and Windows" ;)

**Comment by FrenchBen** [comment]
@mchiang0610 Could you update?

**Comment by thaJeztah** [comment]
Looks like Linux needs to be added now as well :wink:

**Comment by FrenchBen** [comment]
@thaJeztah The code supports it, but we need to add an installer for it :sweat_smile:

**Comment by james-nesbitt** [comment]
@FrenchBen not sure if "having an installer" is really a requirement for adding Linux, as there are some 4-5 different install approaches.  I guess you'd like to have an rpm, and a deb before advertising linux?

Incidentally, I run kitematic from the Arch AUR https://aur.archlinux.org/packages/kitematic/

**Comment by FrenchBen** [comment]
That's correct an RPM or PKG would be nice - Something a bit cleaner than the current clone+compile. 
I do like the Arch Aur setup, just need to automate it for better integration and so that his PKG build is instead part of Grunt.

If you look at the arch linux binary he created, it has:
```js
// TODO : Add 'copy:linux' when will do the packaging
grunt.registerTask('release', ['clean', 'babel', 'less', 'copy:dev', 'electron:linux']);
```
